### DISCUSSION TITLE
Android TabbedBar index

### DISCUSSION THREAD

**Discussion by caspahouzer** [appcelerator/titanium_mobile#12790] [Q&A] [answered]
I have a TabbedBar with no index set. On iOS, it is shown correctly, on Android the first label is selected in frontend. How can I avoid that?

iOS:
![IMG_22BA85A2A53F-1](https://user-images.githubusercontent.com/1126641/117643327-921d6280-b188-11eb-8834-05085d888b73.jpeg)

Android:
<img width="530" alt="Bildschirmfoto 2021-05-10 um 12 07 55" src="https://user-images.githubusercontent.com/1126641/117643363-9a759d80-b188-11eb-8ff7-009a51312efd.png">

**Comment by jquick-axway** [comment, marked as answer]
You can't. On Android, a tab must always be selected. If you don't pick a tab, then it will select the 1st tab.

In Titanium 10.0.0, you can use the new `Ti.UI.OptionBar` which lets you set the "index" property to `-1` to remove the current selection. This will do what you want on both Android and iOS. Have a look at #12510 to see what it looks like.

**Reply by caspahouzer** [reply]
Yeah, that works. But changing to SDK 10 has some side effects. `Ti.App.getArguments()` is missing https://github.com/appcelerator/titanium_mobile/discussions/12801
